src/lib/homestore_backend/hs_homeobject.cpp

```diff
@@ -27,6 +29,7 @@ extern std::shared_ptr< HomeObject > init_homeobject(std::weak_ptr< HomeObjectAp
     auto instance = std::make_shared< HSHomeObject >(std::move(application));
     instance->init_homestore();
     instance->init_timer_thread();
+    instance->init_cp();
```

Comment:
when we had checkpoint for homeobject,  there is no need to add the extra timer thread, right?